execution/specExecutor.go

```diff
@@ -299,6 +299,10 @@ func (e *specExecutor) executeScenario(scenario *gauge.Scenario) (*result.Scenar
 			Name:     scenario.Heading.Value,
 			Tags:     getTagValue(scenario.Tags),
 			IsFailed: false,
+			Retries: &gauge_messages.ScenarioRetriesInfo{ 
```

Comment:
could you please add a test for this? See https://github.com/getgauge/gauge/blob/master/execution/specExecutor_test.go#L617 for a reference

Reply:
'execution/specExecutor_test.go/TestExecuteScenarioShoulHaveRetriesInfo'